README.md

```diff
@@ -60,174 +24,102 @@ After you're finished please remove all the comments and instructions!
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 Microverse_first_project <a name="about-project"></a>
+   This is my new project I created it using HTML and CSS
 
 ## 🛠 Built With <a name="built-with"></a>
+1- HTML.
+2- CSS.
```

Comment:
- [ ] Please remove this, as I can see you have added the technologies in the tech-stack section. So you don't need to write it here and it make your `README.md` looks unprofessional too.